### ISSUE TITLE
[UserNotes] In the overview of a subreddit's usernotes, have an New -> Old sort option

### ISSUE THREAD

**Issue by psdtwk** [creesch/reddit-moderator-toolbox#556]
It's currently sorted oldest notes first. Some mods like to read the notes for their subreddits,. An option for newest to oldest make sense in their case.

**Comment by creesch** [comment]
Yup totally does, sort by username might also be nice. Maybe even sort by note type.
